Assets/Scripts/ModelCatalog.cs

```diff
@@ -253,13 +253,7 @@ public void LoadModelsForNewDirectory(string path)
                 if (m_MissingModelsByRelativePath.ContainsKey(relativePath))
                 {
                     ModelWidget.CreateModelsFromRelativePath(
-                        relativePath, null, m_MissingModelsByRelativePath[relativePath], null, null, null);
-                    m_MissingModelsByRelativePath.Remove(relativePath);
-                }
-                if (m_MissingNormalizedModelsByRelativePath.ContainsKey(relativePath))
```

Comment:
This is why you're merging it and not me; I thought it needed both, and left both `if`s in place in my test, just with fixed calls, and once using  `m_MissingModelsByRelativePath` and one with `m_MissingNormalizedModelsByRelativePath` :-)